test/tables.jl

```diff
@@ -56,18 +56,13 @@ Base.propertynames(d::DuplicateNamesColumnTable) = (:a, :a, :b)
 @testset "Tables" begin
     df = DataFrame(a=Int64[1, 2, 3], b=[:a, :b, :c])
 
-    @testset "basics $(nameof(typeof(table)))" for table in [
-        df,
-        eachrow(df),
-        eachcol(df),
-        eachcol(df, true),
-    ]
+    @testset "basics $(nameof(typeof(table)))" for table in [df, eachrow(df), eachcol(df)]
         @test Tables.istable(table)
         @test Tables.rowaccess(table)
         @test Tables.columnaccess(table)
         @test Tables.schema(table) === Tables.Schema((:a, :b), Tuple{Int64, Symbol})
         @test Tables.schema(table) == Tables.schema(Tables.rows(table)) == Tables.schema(Tables.columns(table))
-        @test @inferred(Tables.materializer(table)(Tables.columns(table))) isa typeof(table)
+        @test @inferred(Tables.materializer(table)(Tables.columns(table))) isa typeof(df)
```

Comment:
```suggestion
        @test @inferred(Tables.materializer(table)(Tables.columns(table))) isa DataFrame
```

Reply:
A small fix. I would pass `DataFrame` explicitly. Also I propose to make an outer loop passing through `[df, view(df, :, :)]` vector. The point is that `Tables.materializer` produces `DataFrame` also for `SubDataFrame`. I think we should test for this here (actually this is relevant wrt. our earlier discussion, as `SubDataFrame` is not resizable).